config/manifests/bases/cert-manager-operator.clusterserviceversion.yaml

```diff
@@ -2,7 +2,6 @@ apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
   annotations:
-    alm-examples: '[]'
```

Comment:
You can still keep this. It will get over written.

Reply:
Okay, done 👍🏻